Cargo.lock

```diff
@@ -8558,6 +8579,15 @@ dependencies = [
  "uuid",
 ]
 
+[[package]]
+name = "oxide-tokio-rt"
+version = "0.1.0"
+source = "git+https://github.com/oxidecomputer/oxide-tokio-rt?rev=bd1b595a32a026163486e09fb785dbdb5fb16081#bd1b595a32a026163486e09fb785dbdb5fb16081"
+dependencies = [
+ "tokio",
+ "tokio-dtrace",
+]
```

Comment:
Could I convince you to put this on crates.io? Checking out a git repo adds notable latency compared to downloading a simple crate tarball.

Reply:
Sure, we can do that. I had figured that for oxide-internal stuff, we preferred to use Git deps, but that was mostly just because we do it for a bunch of other crates.